sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -2,7 +2,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.protobuf.TextFormat;
-import com.mesosphere.sdk.scheduler.recovery.RecoveryPlanManagerFactory;
+import com.mesosphere.sdk.scheduler.recovery.*;
```

Comment:
Can this move back to qualified imports? It makes code easier to read and the move from qualified imports to unqualified seems to confuse diff algorithms.